tests/integration_tests/modules/test_write_files.py

```diff
@@ -50,8 +50,8 @@ class TestWriteFiles:
     @pytest.mark.parametrize(
         "cmd,expected_out", (
             ("file /root/file_b64", ASCII_TEXT),
-            ("md5sum </root/file_binary", "3801184b97bb8c6e63fa0e1eae2920d7"),
-            ("sha256sum </root/file_binary", (
+            ("md5sum /root/file_binary", "3801184b97bb8c6e63fa0e1eae2920d7"),
+            ("sha256sum /root/file_binary", (
```

Comment:
Pipes and redirection are still gonna be weird after this change (as is always the case when using sudo).

Pycloudlib currently does a `sh -c <command>` to any commands passed to it via `execute`. We could perhaps add a sudo kwarg there, though I'm not sure if that's necessary or better than this change.